rust/cubestore/cubestore/src/metastore/mod.rs

```diff
@@ -2670,7 +2757,7 @@ impl RocksMetaStore {
 
             if !taken[i] {
                 taken[i] = true;
-                index_columns.push(seq_column.clone().replace_index(index_columns.len()));
+                index_columns.push(seq_column.replace_index(index_columns.len()));
```

Comment:
Because the `replace_index` method looks like this.
```
pub fn replace_index(&self, column_index: usize) -> Column {
    Column {
        name: self.name.clone(),
        column_type: self.column_type.clone(),
        column_index,
    }
}
```
Cloning of the newly created object is clearly unnecessary